homework/home3.2/Test/TriangleTest.java

```diff
@@ -0,0 +1,101 @@
+import org.example.Triangle;
+import org.junit.jupiter.api.Test;
+import static org.junit.jupiter.api.Assertions.*;
+
+public class TriangleTest {
+
+    @Test
+    public void testArea_3_4_5() {
+        Triangle triangle = new Triangle(3, 4, 5);
+        double s = (3 + 4 + 5) / 2.0;
+        double expected = Math.sqrt(s * (s - 3) * (s - 4) * (s - 5));
+        assertEquals(expected, triangle.area(), 1e-9);
+    }
+
+    @Test
+    public void testArea_1_1_1() {
+        Triangle triangle = new Triangle(1, 1, 1);
+        double s = (1 + 1 + 1) / 2.0;
+        double expected = Math.sqrt(s * (s - 1) * (s - 1) * (s - 1));
+        assertEquals(expected, triangle.area(), 1e-9);
+    }
+
+    @Test
+    public void testArea_5_12_13() {
+        Triangle triangle = new Triangle(5, 12, 13);
+        double s = (5 + 12 + 13) / 2.0;
+        double expected = Math.sqrt(s * (s - 5) * (s - 12) * (s - 13));
```

Comment:
it is better to state the final result of calculation instead of duplication of current implementation cause in general internal formula may be changed while expected value will be the same